concent-vm/helper-scripts/concent-reset.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash -e
+
+# Confirm executing script
+read -p "Are you sure about execute this script? It's destroy databases, queues and other components like: Geth etc. [y/n]" -n 1 -r
```

Comment:
Please reword: `Are you sure you want to remove all databases, queue content and any other content created by the application (except blockchain data)? [y/n] `

Reply:
Done.